.github/workflows/code-validation.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test: [lint, unit-test, csi-compliance-test, controller-integration-test, validate-helm]
+        test: [precommit, lint, unit-test, csi-compliance-test, controller-integration-test, validate-helm]
```

Comment:
This is for double-checking in case someone doesn't check while committing (including myself and agents)